libs/designer/src/lib/core/actions/bjsworkflow/operationdeserializer.ts

```diff
@@ -465,7 +466,7 @@ const initializeOutputTokensForOperations = (
           nodesWithData[operationId]?.settings
         )
       );
-    } catch (e) {
+    } catch {
```

Comment:
not logging the exception?

Reply:
we had changed our tsconfigs recently so that any unused variables error, this code I didn't change, but wanted to pass our commit

Reply:
Yeah that's totally okay but I meant we should be logging the exception/error here. Will help us debug in cases where it fails